jsonexample/test/simple_object_unit_test.dart

```diff
@@ -0,0 +1,196 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:jsonexample/built_value/built_simple_object.dart';
+import 'package:jsonexample/built_value/built_value_serializers.dart';
+import 'package:jsonexample/dart_convert/converted_simple_object.dart';
+import 'package:jsonexample/json_serializable/serializable_simple_object.dart';
+
+void main() {
+  const Map<String, dynamic> typicalObjectJson = {
+    'aString': 'Blah, blah, blah.',
+    'anInt': 1,
+    'aDouble': 1.0,
+    'aListOfStrings': ['one', 'two', 'three'],
+    'aListOfInts': [1, 2, 3],
+    'aListOfDoubles': [1.0, 2.0, 3.0]
+  };
+
+  const Map<String, dynamic> emptyListJson = {
+    'aString': 'Blah, blah, blah.',
+    'anInt': 1,
+    'aDouble': 1.0,
+    'aListOfStrings': [],
+    'aListOfInts': [],
+    'aListOfDoubles': []
+  };
+
+  const Map<String, dynamic> unexpectedPropertyjson = {
+    'aString': 'Blah, blah, blah.',
+    'anInt': 1,
+    'aDouble': 1.0,
+    'aListOfStrings': ['one', 'two', 'three'],
+    'aListOfInts': [1, 2, 3],
+    'aListOfDoubles': [1.0, 2.0, 3.0],
+    'unexpectedProperty': 'Whoops!'
+  };
+
+  group('ConvertedSimpleObject unit tests', () {
+    test('Typical object', () {
```

Comment:
[nit] The concatenated message should ideally read like a sentence. For example: "ConvertedSimpleObject deserializes a typical object".

```
group('ConvertedSimpleObject', () {
  test('deserializes a typical object', () {
```

Reply:
Rewrote all of 'em.